intro-to-R-tidyverse/03-intro_to_tidyverse.Rmd

```diff
@@ -84,11 +84,20 @@ If we want to more pointedly look for "results" in our working directory we can
 dir.exists("results")
 ```
 
-If the above says `FALSE` that means we will need to create a `results` directory using the function `dir.create()`.
+If the above says `FALSE` that means we will need to create a `results` directory.
+We've previously seen that we can make directories in R using the base R function `dir.create()`.
+But, as we've also seen, this function will throw an error if you try to create a directory that already exists, which can be frustrating if you are re-running code!
+A different option is to use the [`fs`](https://fs.r-lib.org/) package, which provides functions for you to interact with your computer's file system, and _bonus_ it's guaranteed to work on any kind of operating system (Max, Linux, PC, etc.)!
```

Comment:
Is there any reason that we would expect dir.create() not to work on any operating sytem? This seems like FUD.
```suggestion
A different option is to use the [`fs`](https://fs.r-lib.org/) package, which provides functions for you to interact with your computer's file system with a more consistent behavior than the base R functions.
```